File: MPI/mpicheck/tests/functions/4.0/MPI_Group_from_session_pset.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Session var_0;
    const char *var_1;
    MPI_Group *var_2;
    int ret;
    /* calls */
    ret = MPI_Group_from_session_pset(var_0, var_1, var_2);
    ret = PMPI_Group_from_session_pset(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/4.0/MPI_Group_from_session_pset.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       CHARACTER*(10) var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_group_from_session_pset(var_0, var_1, var_2, var_3)
       call pmpi_group_from_session_pset(var_0, var_1, var_2, var_3)
        end program main
    